web-registry/src/components/organisms/RegistryLayout.tsx

```diff
@@ -97,9 +96,7 @@ const RegistryLayout: React.FC<React.PropsWithChildren<unknown>> = () => {
           <Box display="flex" justifyContent="center" alignItems="center">
             {chainId && loaded && wallet?.address && disconnect && (
               <UserMenuItem
-                address={
-                  isMobile ? truncate(wallet?.address) : wallet?.shortAddress
-                }
+                address={truncate(wallet?.address)}
```

Comment:
A small request from @erikalogie to have a consistent address truncating between desktop and mobile.